addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -506,7 +506,7 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                 category: 'Website',
                 name: 'Alert',
                 priority: 100,
-                description: 'Insert an alert snippet.',
+                description: 'Insert an alert snippet',
```

Comment:
Aren't all those in `website` missing the `_t()`?

Reply:
Indeed. Working on it, I'll make a new PR targeting 15.0.

Reply:
Done in PR https://github.com/odoo/odoo/pull/103791